protocol-samples/http-client-to2-sample/src/main/java/org/fidoalliance/fdo/sample/To2ClientApp.java

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 Intel Corporation
 // SPDX-License-Identifier: Apache 2.0
 
-package org.fido.iot.sample;
```

Comment:
Check line 215 , Consider changing from "/home/fido" -> "/home/fdo"

Reply:
Updated in a1d9a7